test/check-machines-create

```diff
@@ -911,6 +928,17 @@ class TestMachinesCreate(VirtualMachinesCase):
             elif self.os_name:
                 b.click("#os-select-group > div button")
                 b.click("#os-select li:contains('{0}') button".format(self.os_name))
+                b.wait_attr("#os-select-group input", "value", self.os_name)
+                if (self.expected_os_name and
+                    self.expected_location and
+                    self.sourceType == 'url'):
+                    b.set_input_text("#source-url", self.expected_location)
+                    b.wait_attr("#os-select-group", "data-loading", "false")
+                    b.wait_attr("#os-select-group input", "value", self.expected_os_name)
+
+                    b.switch_to_top()
```

Comment:
This does not belong here. If you want to check that the page did not crash please introduce a function and run it after the fill method.